x-pack/legacy/plugins/siem/public/components/ml_popover/ml_popover.test.tsx

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import {
+  getConfigTemplatesToInstall,
+  getJobsToDisplay,
+  getJobsToInstall,
+  JobDetail,
+  JobDetailsHeader,
+  MlPopover,
+  MlPopoverTitle,
+} from './ml_popover';
+import {
+  mockConfigTemplates,
+  mockEmbeddedJobIds,
+  mockInstalledJobIds,
+  mockJobsSummaryResponse,
+} from './__mocks__/api';
+
+describe('MlPopover', () => {
+  describe('MlPopover', () => {
+    test('showing a popover on a mouse click', () => {
+      const wrapper = mount(<MlPopover />);
+      wrapper
+        .find('[data-test-subj="integrations-button"]')
+        .first()
+        .simulate('click');
+      wrapper.update();
+      expect(wrapper.find('[data-test-subj="ml-popover-contents"]').exists()).toEqual(true);
+    });
+  });
+
+  describe('MlPopoverTitle', () => {
+    test('renders correctly against snapshot', () => {
+      const wrapper = shallow(<MlPopoverTitle />);
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+  });
+
+  describe('JobDetailsHeader', () => {
+    test('renders correctly against snapshot', () => {
+      const wrapper = shallow(<JobDetailsHeader />);
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+  });
+
+  describe('JobDetail', () => {
+    const onJobStateChangeMock = jest.fn();
+
+    test('renders correctly against snapshot', () => {
+      const wrapper = shallow(
+        <JobDetail
+          key={'job.title'}
+          jobName={'job.title'}
```

Comment:
Optional:

You should be able to do this right?
```ts
jobName="job.title"
```

I didn't see a linter rule for it we can turn on though:
https://github.com/yannickcr/eslint-plugin-react#jsx-specific-rules

which is a bit of bummer. Let me know if I'm missing a rule, I do this sometimes too and it's not needed with attributes. Would be nice to be more strict in some of these cases at some point soon 🤞